src/reducers/index.ts

```diff
@@ -19,10 +19,10 @@ export interface Action {
   // In contrast, actions that start with "update" will update mappigns
   // (using mergeMaps below).
   setGamesList?: GameInfo[];
-  updateGameSpecs?: GameSpecs;
+  updateGameSpecs?: GameSpecs; 
   setMatchesList?: MatchInfo[];
-  setCurrentMatchIndex?: number; // an index in matchesList
-  updateMatchIdToMatchState?: MatchIdToMatchState;
+  setCurrentMatchIndex?: number; // an index in matchesList   
+  updateMatchIdToMatchState?: MatchIdToMatchState;  
```

Comment:
In line 53 below, can you add a check that:

// Ensure UserIdsAndPhoneNumbers have two mappings that are exactly the reverse of each other.

And have a test that checks that.